src/components/Filter.tsx

```diff
@@ -0,0 +1,140 @@
+import styled from '@emotion/styled'
+import { useState } from 'react'
+import { FontVariant, Color, Radius } from '@/app/theme'
+import { IoChevronDownOutline, IoChevronUpOutline } from 'react-icons/io5'
+
+const Select = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+`
+
+const SelectName = styled.span<{ filtered: boolean }>`
+  margin-left: 4px;
+  ${FontVariant.body_sm}
+  width: fit-content;
+  margin-right: 6px;
+  color: ${props => (props.filtered ? Color.orange900 : Color.gray700)};
+  white-space: nowrap;
+`
+
+const SelectBody = styled.div`
+  position: relative;
+  min-width: 230px;
+  width: 100%;
+  width: fit-content;
+
+  ${FontVariant.body_md}
+
+  box-sizing: border-box;
+`
+
+const Value = styled.span<{ filtered: boolean }>`
+  display: flex;
+
+  overflow: hidden;
+
+  border: 1px solid ${props => (props.filtered ? Color.orange900 : Color.gray300)};
+  border-radius: ${Radius.md};
+  padding: 6px 16px 6px 12px;
+
+  cursor: pointer;
+
+  vertical-align: middle;
+  justify-content: space-between;
+  color: ${props => (props.filtered ? Color.orange900 : Color.gray700)};
+
+  white-space: nowrap;
+  text-overflow: ellipsis;
+
+  &:hover {
+    // background-color: ${props => (props.filtered ? Color.orange50 : Color.gray50)};
```

Comment:
remove commented code

Reply:
Here's the update!: [5339c3b](https://github.com/kixlab/website/pull/13/commits/5339c3bba60ecac1047e3959299a8de0a1fd274b)